components/casemap/src/casemapper.rs

```diff
@@ -146,16 +146,16 @@ impl CaseMapper {
     /// as a `LanguageIdentifier` (usually the `id` field of the `Locale`) if available, or
     /// `Default::default()` for the root locale.
     ///
-    /// This function performs legacy head adjustment behavior when [`HeadAdjustment::Adjust`] is set. See
+    /// This function performs "adjust to cased" head adjustment behavior when [`HeadAdjustment::Adjust`] is set. See
```

Comment:
"head adjustment" is confusing. I have never heard "head" used in discussion of titlecasing.
How about "titlecasing index adjustment" or similar?

Reply:
We've been calling this HeadAdjustment and TailCasing which I think has a nice symmetry. "Index adjustment" feels more internal, I think "head" is quite clear personally.

Reply:
Bikeshed:

- Head/Tail
- Lead/Trail
- Start/Remainder